### PR TITLE
[MH-000] Tasks file

### PR DESCRIPTION
**Pull request by VasckMe**
First pull request into develop.
Here you can find adding tasks and some rules about them in TASKS.md file

### CODE REVIEW

File: TASKS.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] [001] Write in README commit rules
+- [ ] [002] Write in README branch rules
+- [ ] [003] Add dependencies for frameworks + architecture folders and files
+- [ ] [004] Implement function to read json dataset with given dataset name
+- [ ] [005] Add logic with to divide dataset text into words, classes and documents
+- [ ] [006] Write logic for saving words and classes into binary files in folder ‘saved_data’ 
+- [ ] [007] Make binary code from input dataset data
+- [ ] [008] Write code for creating instance (model) with Sequential and configure it
+- [ ] [009] Compile and save ML Model. Add print in the end of the code to show the end of the compiling
+
```
Comment (Koris493) [inline TASKS.md:14]: W jakiejkolwiek linijce test w komentarzu

**Comment by AoDem0** [comment]
>:)

**Comment by Koris493** [comment]
ok, dobrze, test smile

File: TASKS.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+> [!TIP]
+> If you need to add, edit or remove task, here is the examples
+
```
Comment (AoDem0) [inline TASKS.md:17]: tipsy :3

File: TASKS.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] not finished task
+- [x] completed task
+
+**case 1**: you need to add task in 006 place
+**solution**: please, write [006a] code for it
+
+**case 2**: you need to change task
+**solution**: strikethrough previous task and write new with the same code:
+- ~~[006] Add logic with to divide dataset text into words, classes and documents~~
+- [006] Make refactoring
```
Comment (AoDem0) [inline TASKS.md:29]: 132
@VasckMe

**Review by AoDem0 — CHANGES_REQUESTED** [review]

**Review by Koris493 — CHANGES_REQUESTED** [review]
Comment

**Review by AoDem0 — APPROVED** [review]

**Review by Koris493 — APPROVED** [review]
💣

**Pull request merged**